### PR TITLE
RESOLVE: Issue #5 Tests

### PR DESCRIPTION
**Pull request by strictly-scarce**
Resolves Issue1

### CODE REVIEW

File: test/vBTC_test.js [hunk 1]
```diff
@@ -58,16 +61,40 @@ contract('virtualBitcoin', function(accounts) {
...
+    // unauthorised transfer
+    let tx0 = await truffleAssert.reverts(coin.transfer(acc0, '1000000000000000000', { from: acc1 }))
 
+    // authorised transfer
     let tx1 = await coin.transfer(acc1, '1000000000000000000', { from: acc0 })
-    console.log(tx1);
 
     let bal0 = await coin.balanceOf(acc0);
-    assert.equal(bal0, 20999999_000000000000000000, "correct coin bal");
+    assert.equal(bal0, 20999999000000000000000000, "correct coin bal");
```
Comment (strictly-scarce) [inline test/vBTC_test.js]: Why did you need to do this?

Comment (KIC8462852-sol) [reply test/vBTC_test.js]: because it ran an error

**Comment by KIC8462852-sol** [comment]
Resolved issue #5
